test/sol/voracle/TestLag.sol

```diff
@@ -13,24 +13,29 @@ contract TestLag {
     function beforeEach() public {
         mockMedianizer = new MockMedianizer(10);
         mockMedianizer2 = new MockMedianizer(11);
-        lag = new Lag(address(this), address(0));
+        lag = new Lag();
+        lag.initialize(address(this), address(0));
```

Comment:
```suggestion
        lag.initialize(address(this), address(mockMedianizer));
```

Reply:
we set mock medianizer on the case we really need

Reply:
okay